client/web/web.go

```diff
@@ -1108,6 +1140,285 @@ func (s *Server) servePostRoutes(ctx context.Context, data postRoutesRequest) er
 	return err
 }
 
+type serveItem struct {
+	Target      serveTarget      `json:"target"`
```

Comment:
oh, this reminds me of all the rounds of discussion about terminology.  In this context, I would typically think of `target` and `destination` as synonyms. Hmmm, is that what we settled on?  If so, I guess it's fine 🤷🏻  

Maybe if Destination were DestinationOptions?  But that gets to be pretty long. :-\